core/gui/src/app/app-routing.module.ts

```diff
@@ -58,39 +58,46 @@ if (environment.userSystemEnabled) {
       },
       {
         path: "user",
-        canActivate: [AuthGuardService],
```

Comment:
Move the search component to the same level as `user` instead of setting AuthGuard for each child.